CHANGELOG.md

```diff
@@ -4,6 +4,46 @@ This document includes a curated changelog for each release. We also publish a c
 a [GitHub release](https://github.com/nginx/nginx-gateway-fabric/releases), which, by contrast, is auto-generated
 and includes links to all PRs that went into the release.
 
+## Release 1.6.0
+
+_January 15, 2025_
+
+FEATURES:
+
+- Add UpstreamSetttingsPolicy. [2941](https://github.com/nginx/nginx-gateway-fabric/pull/2941)
+- Add path support for RequestRedirect Filter. [2979](https://github.com/nginx/nginx-gateway-fabric/pull/2979)
+
+BUG FIXES:
```

Comment:
There are a couple release notes to be added from https://github.com/nginx/nginx-gateway-fabric/pull/2897. I think two bug fixes and one feature.